docusaurus.config.ts

```diff
@@ -12,11 +12,11 @@ const config: Config = {
   url: "https://quantstack.net/",
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: "/",
+  baseUrl: "/quantstack.github.io/",
 
   // GitHub pages deployment config.
   // If you aren't using GitHub pages, you don't need these.
-  organizationName: "/QuantStack/", // Usually your GitHub org/user name.
+  organizationName: "/HaudinFlorence/", // Usually your GitHub org/user name.
```

Comment:
Looks like these two changes could be reverted?

Reply:
Indeed. I have changed it as well as the `baseUrl` that has to be `"/"`